deel-2-flexbox-layout/opdracht-4/styles/styles.css

```diff
@@ -1 +1,61 @@
-/*Schrijf hier jouw CSS*/
\ No newline at end of file
+/* -- Flex-row op body -- */
+
+.flex-body-row {
+  display: flex;
+  min-height: 100vh;
+}
+
+/* -- Control width NAV and A-SIDE  -- */
+
+.flex-row-col-1, .outer-col-aside {
+  flex-basis: 250px;
+  flex-grow:0;
+  flex-shrink: 0;
+}
+
+/* -- Control height HEADER and FOOTER  -- */
+
+.flex-col-header, .flex-col-footer {
+  flex-basis:130px;
+  flex-grow: 0;
+  flex-shrink: 0;
+}
+
+
+.flex-row-col-2 {
+  flex-grow:1;
+}
+
+/* -- Flex-column op main */
+
+.flex-col-main {
+  display: flex;
+  flex-direction: column;
+}
+
+.inner-row {
+  flex-grow:1;
+}
+
+/* -- Flex-row op inner-row -- */
+
+.flex-row-content {
+  display: flex;
+}
+
+.inner-col {
+  flex-grow: 1;
+}
+
+/* -- Flex-column op inner-col -- */
+
+.flex-col-content {
+  display: flex;
+  flex-direction: column;
+}
+
+.flex-col-article {
+  flex-basis:auto;
+  flex-grow:1;
+}
+
```

Comment:
elke spatie en regel heb je zo goed gezet in elke opdracht, ik heb een voor een allemaal gecheckt. geen enkele fout.

Reply:
Ben een klein beetje autistisch zeg ik dan altijd. :-)